src/Sulu/Bundle/RouteBundle/Resources/config/doctrine/BaseRoute.orm.xml

```diff
@@ -19,6 +19,11 @@
                 <option name="default">1970-01-01 00:00:00</option>
             </options>
         </field>
+        <field name="changed" type="datetime" nullable="false">
```

Comment:
because the field will be added by auditable interface and the default shouldnt be there.